nullaway/src/main/java/com/uber/nullaway/NullAway.java

```diff
@@ -2014,7 +2018,10 @@ private Description createErrorDescriptionForNullAssignment(
       fix =
           SuggestedFix.prefixWith(
               suggestTree,
-              "@SuppressWarnings(\"" + checkerName + "\")" + config.getAutofixSuppressionComment());
+              "@SuppressWarnings(\""
```

Comment:
Just a formatting change?

Reply:
I just added the space at the end, GJF did the rest :)